myia/xtype.py

```diff
@@ -318,3 +318,41 @@ class External(typing.Generic[T]):
     """Represents a type external to Myia (essentially invalid)."""
 
     obj: T
+
+
+__all__ = [
+    'Bool',
+    'Dict',
+    'EnvType',
+    'ExceptionType',
+    'External',
+    'Float',
+    'Int',
+    'NDArray',
+    'Nil',
+    'NotImplementedType',
+    'Number',
+    'Object',
+    'String',
+    'SymbolicKeyType',
+    'T',
```

Comment:
Not T, TypeMeta, as_frozen and maybe not Type too.

Reply:
`TypeMeta` is used elsewhere.